membership/management/commands/csvbills.py

```diff
@@ -179,7 +179,10 @@ def _get_row(self, row):
 
 def row_to_payment(row):
     try:
-        p = Payment.objects.get(transaction_id__exact=row['transaction'])
+        # Search first for from already imported rows
```

Comment:
Should we add a comment here [csvbills#L112-L122 BillDictReader.next()](https://github.com/annttu/sikteeri/blob/335ac6b22f7cccdd72b8d38a6541305ca85cb87c/membership/management/commands/csvbills.py#L112-L122) that row_to_payment is sensitive to changes in the data cleaning done in the next function?

Meaning if something was imported with old version, this would fail in later version as the normalization will be different.